meeteval/wer/wer/time_constrained.py

```diff
@@ -5,6 +5,8 @@
 import typing
 from dataclasses import dataclass, replace
 
+import transphone
```

Comment:
Can you use a lazy import? We want to keep the mandatory dependencies for the core code small.